New package: EphemerisSources v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: EphemerisSources
- Repository: https://github.com/cadojo/EphemerisSources.jl
- Created by: @cadojo
- Version: v0.1.0
- Commit: cdafd5c3451bf649ccd5a6e4dd5744cd55c123af
- Reviewed by: @cadojo
- Reference: https://github.com/cadojo/EphemerisSources.jl/commit/cdafd5c3451bf649ccd5a6e4dd5744cd55c123af#commitcomment-141430876
- Description: Access HORIZONS and SPICE ephemeris sources with idiomatic Julia.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6f443a0e793bbef1bc3e5b9e66b69c6f6fcf36f51900c212e45018d829e98d744db49badc1590e5e9e0a392b478b5136ce8ec14d3432997c346ef6cc4b47ce211f99793faa326e42755f54de21bab389fc59969a96fdf95f70ca5956c430324929fe7b398fc9595165ff816cdfecace8ebf691d4d379f0f0646a29ee0419fe90b8a8b890f4f91ea7c31364d4ef41e4ff2a216a9e684af9e04ed3e503a3b5d98191f924c3079d362ef6b7a9235d922ef98 -->

cadojo commented:
This should not be merged. This was my attempt to release a sub package. Apologies for the confusion.

JuliaRegistrator committed:
New package: EphemerisSources v0.1.0

UUID: 858900df-f7ae-402a-aa8c-3e4ef85aa271
Repo: https://github.com/cadojo/EphemerisSources.jl.git
Tree: 33cc6bb5bedccc89dbc28ab0ed3a7813ea55b575

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
